packages/core/src/components/Accordion.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import {
+  View,
+  StyleSheet,
+  StyleProp,
+  TextStyle,
+  Pressable,
+} from "react-native";
+import Text from "./Text";
+import { withTheme } from "../theming";
+import type { IconSlot } from "../interfaces/Icon";
+import type { Theme } from "../styles/DefaultTheme";
+import { colorTypes } from "@draftbit/types";
+
+import {
+  COMPONENT_TYPES,
+  createTextProp,
+  createNumberProp,
+  createIconProp,
+  createColorProp,
+} from "@draftbit/types";
+
+type Props = {
+  openColor: colorTypes;
```

Comment:
```suggestion
  openColor: string;
```

all of these can be `string`, not `colorTypes`. We've realized that the basic string makes it more flexible with other components.

Reply:
done